m4/ax_cxx_compile_stdcxx.m4

```diff
@@ -0,0 +1,982 @@
+# ===========================================================================
```

Comment:
I'd love to, but I can't seem to find the version that makes it work, or the actual name of the macro...  :(